### ISSUE TITLE
Model baking not working when table contains underscore

### ISSUE THREAD

**Issue by gnh1201** [cakephp/bake#283]
The same problem this issue. Can I solve this?

ex)
DATABASE: shop
TABLENAME: sh_cart
error message: shop.cart ( it is not shop.sh_cart ) dose not exists.

https://github.com/cakephp/bake/issues/132

**Comment by markstory** [comment]
What other tables do you have in that database? I have a few apps that have tables with `_` in them and bake works fine. Perhaps bake is getting confused with associations?

Also how are you running bake when you run into this issue?

**Comment by gnh1201** [comment]
I could check difference after run bake when use this case:

Database: myapp
- myapp.sh_cart or myapp.shcart -> (after bake) myapp.carts
- myapp.hm_post or myapp.hmpost -> (after bake) myapp.posts
- myapp.hm_post_history or myapp.hmposthistory -> (after bake) myapp.post_histories
- myapp.pf_orgranization or myapp.pforgranization -> (after bake) myapp.orgs

I do not know why occurred this difference. Does exists reserved words in Bake?

**Comment by josegonzalez** [comment]
@gnh1201 What command are you running to execute bake for these tables? As well, can you paste the version of bake and cakephp you have installed?

**Comment by gnh1201** [comment]
version: 3.3.9
commands:
```
bin/cake bake model all
bin/cake bake controller all
bin/cake bake fixture all
bin/cake bake template all
```

**Comment by markstory** [comment]
> Does exists reserved words in Bake?

No there aren't any reserved words in bake as far as table/model names are concerned.

**Comment by gnh1201** [comment]
Note: #286

**Comment by gnh1201** [comment]
I fixed this issue: #283
